service/src/main/java/uk/gov/hmcts/reform/fpl/service/ManageDocumentService.java

```diff
@@ -249,6 +274,24 @@
         return setDateTimeUploadedOnSupportingEvidence(currentSupportingDocuments, previousSupportingDocuments);
     }
 
+    //Separate collection based on idam role (only show users their own documents)
+    private List<Element<SupportingEvidenceBundle>> getUserSpecificSupportingEvidenceBundle(
+        List<Element<SupportingEvidenceBundle>> bundleList) {
+        UserDetails userDetails = idamClient.getUserDetails(requestData.authorisation());
+        boolean isHmctsUser = userDetails.getRoles().stream().anyMatch(UserRole::isHmctsUser);
+
```

Comment:
maybe these two should be part of a new service called `UserService` or `UserDetailsService`